Source/FieldSolver/WarpXPushFieldsEM.cpp

```diff
@@ -76,6 +76,7 @@ WarpX::EvolveB (int lev, PatchType patch_type, amrex::Real dt)
         const Box& tbx  = mfi.tilebox(Bx_nodal_flag);
         const Box& tby  = mfi.tilebox(By_nodal_flag);
         const Box& tbz  = mfi.tilebox(Bz_nodal_flag);
+        const Real xmin = mfi.tilebox().smallEnd(0)*dx[0];
```

Comment:
Could you add a comment here, mentioning that, in practice, this used only by the underlying PICSAR kernel in the case of `rz` geometry? (in which case it corresponds to `rmin`)